.github/dependabot.yml

```diff
@@ -36,4 +36,50 @@ updates:
   - dependency-name: "google.golang.org/grpc"
   labels:
   - "ok-to-test"
-  - "kind/cleanup"
\ No newline at end of file
+  - "kind/cleanup"
+# Go - release 1.7
+- directory: "/"
```

Comment:
where's the part in this definition that declares "security fixes only"? (or is that actually not possible?)

Reply:
We achieve that by setting `open-pull-requests-limit` to `0` to prevent version updates for a given package-ecosystem [as mentioned here](https://docs.github.com/en/code-security/dependabot/dependabot-security-updates/configuring-dependabot-security-updates#overriding-the-default-behavior-with-a-configuration-file)

----

- For release-1.7 https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/3056/files#diff-dd4fbda47e51f1e35defb9275a9cd9c212ecde0b870cba89ddaaae65c5f3cd28R61-R62
- For release 1.6 https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/3056/files#diff-dd4fbda47e51f1e35defb9275a9cd9c212ecde0b870cba89ddaaae65c5f3cd28R84-R85

Reply:
could you please add that as a comment above `open-pull-requests-limit: 0`? We might not remember why it's there later. 

LGTM otherwise

Reply:
Absolutely!